test_vm/src/lib.rs

```diff
@@ -71,10 +72,126 @@ use std::collections::HashMap;
 use std::error::Error;
 use std::fmt;
 use std::ops::Add;
+use util::apply_ok_;
+use util::pk_addrs_from;
 
+pub mod bench;
+pub mod executor;
 pub mod util;
 
-pub struct VM<'bs> {
+pub trait VM {
```

Comment:
This is not really final, just covered the minimal surface area so far in order to get the `change_owner_test` to run